ci/Jenkinsfile.combined

```diff
@@ -1,4 +1,4 @@
-library 'status-jenkins-lib@v1.3.3'
+library 'status-jenkins-lib@creds/opensea-api-key'
```

Comment:
Please update the references to use the tag.

Reply:
yep, updated